community-roles.md

```diff
@@ -113,6 +114,24 @@ The following apply to all assets across the Flux org:
 - Defining the scope of each Git repository
 - Resolving escalated decisions when Maintainers responsible are blocked
 
+### Security Team member
+
+Security Team members are listed in the [SECURITY.md file](https://github.com/fluxcd/.github/blob/main/SECURITY.md#security-team).
+
+Members of this team handle security issues for the Flux projects. It is essential for us to deal with security-related concerns responsibly and follow the high standards of the security community as a whole.
+
+**Defined by:** entry in [SECURITY.md file](https://github.com/fluxcd/.github/blob/main/SECURITY.md#security-team).
+
+**Responsibilities and Privileges:**
+
+The following apply to all assets across the Flux org:
+
+- All reports are thoroughly investigated by the Security Team.
+- Any vulnerability information shared with the Security Team will not be shared with others unless it is necessary to fix the issue. Information is shared only on a need to know basis.
+- As the security issue moves through the identification and resolution process, the reporter will be notified.
+- Additional questions about the vulnerability may also be asked of the reporter.
+- Security Team members have a duty of care to the uphold of the Security embargo policy.
```

Comment:
@dholbach on second thought, perhaps this section can just link to the Handling section of SECURITY.md, since most is repeated, and just add this new embargo item to the list there?

Reply:
So we only have to edit this in one place

Reply:
Hum. Yes. On the one hand I know it is copied from there. On the other hand did I want to give readers ab explanation of what the team does...